server.py

```diff
@@ -38,6 +38,22 @@ def add_note():
 
     return jsonify(response)
 
+@app.route('/notes/edit/<id>', methods=['POST'])
```

Comment:
When I use PUT I get an error messages: 405 Method not allowed. I have googled and don't see a resolution to this issue. Requires further research.